dynamodb/src/main/scala/akka/stream/alpakka/dynamodb/impl/DynamoClientImpl.scala

```diff
@@ -22,12 +24,12 @@ class DynamoClientImpl(
   override protected val service = "dynamodb"
   override protected val defaultContentType =
     ContentType.Binary(MediaType.customBinary("application", "x-amz-json-1.0", NotCompressible))
-  override protected implicit val ec = system.dispatcher
+  override protected implicit val ec: ExecutionContextExecutor = system.dispatcher
 
   override protected val connection: AwsConnect =
     if (settings.port == 443)
-      Http().cachedHostConnectionPoolHttps[AwsRequestMetadata](settings.host)(materializer)
+      Http().cachedHostConnectionPoolHttps(settings.host)
```

Comment:
Explicitly passing the `materializer` here makes the compilation fails. It's strange !